test/json_texture_test.js

```diff
@@ -0,0 +1,28 @@
+var assert = require('assert');
+var configUtils = require('./utils/config');
+var testUtils = require('./utils/test');
+var fs = require('fs');
+var pkg = require('../package.json');
+
+describe('A hash map of texture info, similar to TexturePacker', function () {
+  testUtils.setInfo(configUtils.multipleSprites);
+
+  function assertValidJson() {
+    it('is valid JSON', function () {
+      var result = this.result;
+      assert.doesNotThrow(function () {
+        JSON.parse(result);
+      });
+    });
+  }
```

Comment:
This `assertValidJson` also exists in the JSON test file. I was suggesting to relocate the assertions into `utils/test.js` for reused cose. This is fine though, I will move things around post merge.